ui/src/app/utils/access-utils.tsx

```diff
@@ -290,4 +291,9 @@ export const computeDisplayDates = ({completionEpochMillis, expirationEpochMilli
   );
 };
 
-
+// return ture if user is egligible for registered tier.
+// A user loses tier eligiblity when they are remmoved from institution tier requirement
+export const eligibaleForRegisteredForTier = (tierEligiblities: Array<UserTierEligibility>): boolean => {
+  const rtEligiblity = tierEligiblities.find( t => t.accessTierShortName === AccessTierShortNames.Registered)
+  return rtEligiblity && rtEligiblity.eligible
```

Comment:
In this case, I would use `!!rtEligibility && ...`. Otherwise `rtEligibility` is null, this helper returns null. Callers will expect it to return boolean.